overthrower/overthrower.cpp

```diff
@@ -272,7 +283,8 @@ void* malloc(size_t size)
 #if !defined(__APPLE__)
     if (activated != 0 && pointer != NULL) {
         std::lock_guard<std::mutex> lock(mutex);
-        allocated.insert(pointer);
+        alloc_counter++;
```

Comment:
Ok. But first item you insert below will have number 1 (not 0). Because `alloc_counter` is incremented before inserting to map.